### PR TITLE
DOCS-1600: Add configure a fleet page

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by npentrel — CHANGES_REQUESTED** [review]
Great start - a few things we can improve a little more

File: docs/fleet/configure-a-fleet.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+linkTitle: "Configure a Fleet"
+weight: 40
+type: "docs"
+description: Use fragments to configure many machines at the same time.
+tags: ["fleet management", "cloud", "app"]
+---
+
+If you have multiple identical or similar machines, use a _fragment_ to configure all of the machines at the same time.
+
+Fragments are a way of sharing and managing [machine configurations](/build/configure) across multiple machines.
```
Comment (JessamyT) [inline docs/fleet/configure-a-fleet.md:12]:
```suggestion
Fragments are a way of sharing and managing [machine configurations](/build/configure/) across multiple machines.
```

File: docs/fleet/configure-a-fleet.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+weight: 40
+type: "docs"
+description: Use fragments to configure many machines at the same time.
+tags: ["fleet management", "cloud", "app"]
+---
+
+If you have multiple identical or similar machines, use a _fragment_ to configure all of the machines at the same time.
+
+Fragments are a way of sharing and managing [machine configurations](/build/configure/) across multiple machines.
+When changes are made to the fragment, the Viam app automatically applies the changes to all machines that include the fragment in their config.
```
Comment (npentrel) [inline docs/fleet/configure-a-fleet.md:13]:
```suggestion
When you or one of your collaborators changes the fragment, the Viam app automatically applies the changes to all machines that include the fragment in their config.
```

File: docs/fleet/configure-a-fleet.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Before you create a fragment, you'll need a configuration file.
+Start by [configuring](/build/configure/) one of your identical machines on its **CONFIG** tab in the [Viam app](https://app.viam.com).
+Now you're ready to share that config by creating a fragment.
+
+To create your own private fragment, go to [app.viam.com/fragments](https://app.viam.com/fragments) or click on **Fragments** in the left navigation bar on the [FLEET page](https://app.viam.com/robots).
+
+1. Enter a name for your new fragment and click **Add fragment**.
+2. Add the JSON configuration in the config field.
+   You can get the JSON configuration from the [**CONFIG** tab](/build/configure/#the-config-tab) of your first machine by switching to **Raw JSON** mode.
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/fleet/configure-a-fleet.md:29]: that sounds good too

**Pull request merged**